### DISCUSSION TITLE
ldap2pg cron.d

### DISCUSSION THREAD

**Discussion by sla82** [dalibo/ldap2pg#620] [Q&A]
Hello, i want to run ldap2pg automaticly. But with a file in /etc/cron.d/ldap2pg it doesnt work.  The content of the file is
`* * * * * postgres /usr/bin/ldap2pg  --real &>> /tmp/ldap_debug.log`

What i need to do to run it with cron?

**Comment by bersace** [comment]
What is the content of cron mail and ldap_debug.log ?

**Comment by sla82** [comment]
The ldap_debug.log is empty. I can not find cron mail. The system is a Debain 12.

**Comment by bersace** [comment]
You may need to ask for cron help on stack overflow.
